unity_buy_android/unity_buy_plugin/src/main/java/com/shopify/unity/buy/UnityBuyPlugin.java

```diff
@@ -1,6 +1,8 @@
 package com.shopify.unity.buy;
 
 public class UnityBuyPlugin {
+    private UnityBuyPlugin() { }
```

Comment:
Checkstyle complains that utility classes shouldn't have a default or public constructors. Makes sense considering a class that only has static methods should never be instantiated. 

This class it just a dummy but figured I'd fix it so the test script doesn't fail.